network/common/src/main/java/org/apache/spark/network/TransportContext.java

```diff
@@ -17,13 +17,10 @@
 
 package org.apache.spark.network;
 
-import java.util.List;
-
-import com.google.common.collect.Lists;
 import io.netty.channel.Channel;
 import io.netty.channel.socket.SocketChannel;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+
+import java.util.List;
```

Comment:
the dependencies should be ordered as
1. java lib
2. scala lib
3. 3-rd party lib
4. spark packages